### PR TITLE
Sitemap Index and Splitting into multiple sitemaps

### PR DESCRIPTION
**Pull request by Matzu89**
Implements as described in #49

### CODE REVIEW

**Comment by kuflash** [comment]
@Matzu89 thank you! But TravisCI show some lint errors. Can you fix it? You can run command `npm run lint` for check your files.

File: lib/index.js [hunk 1]
```diff
@@ -104,17 +105,40 @@ class Sitemap {
 	 * @description Convert array of paths to sitemap.
 	 * @param {String} hostname - The root name of your site.
 	 */
-	build(hostname) {
-		this.sitemap = buildSitemap(hostname, this.paths);
+	build(hostname) {		
+		this.hostname = hostname;
+		this.splitted = splitPaths(this.paths, 49999);
```
Comment (kuflash) [inline lib/index.js]: The number of paths must be made an optional parameter. By default it equal 49999. 
Create second argument `options` as object and add property limitCountPaths.
For example:

 ```js
build('https://example.com', { limitCountPaths: 1000 })
```

File: lib/paths-splitter/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * @example
+ * import { pathsSplitter } from 'react-router-sitemap';
+ *
+ * const splitted = pathsSplitter(paths, 49999); // 49999 because of Google sitemap limits
+ */
+
+const pathsSplitter = (paths, size) => paths.map((e, i) => {
+	return (i % size === 0) ? paths.slice(i, i + size) : null;
+}).filter(e => e);
```
Comment (kuflash) [inline lib/paths-splitter/index.js]: Please rename `e to `path` for more readability

**Comment by kuflash** [comment]
@Matzu89 also please add unit tests for new module `paths-splitter`.

**Comment by Matzu89** [comment]
Your request is my demand. Executed, this should do the trick.

**Comment by kuflash** [comment]
@Matzu89 cool!

**Pull request merged**